tests/e2e/package.json

```diff
@@ -27,7 +27,7 @@
     "mocha": "^8.2.1",
     "mochawesome": "^6.2.2",
     "module-alias": "^2.2.2",
-    "prestashop_test_lib": "0.0.12"
+    "prestashop_test_lib": "0.0.14"
```

Comment:
This package should be managed by the open source project no?

Reply:
yes it's still WIP

Reply:
Even if it's in WIP, we can maybe move the repository to this organization.